designsafe/static/scripts/workspace/controllers/application-form.js

```diff
@@ -203,11 +203,11 @@ export default function ApplicationFormCtrl($scope, $rootScope, $localStorage, $
 
             // Calculate processorsPerNode if nodeCount parameter submitted
             if (('nodeCount' in jobData) && !('processorsPerNode' in jobData)) {
-                jobData.processorsPerNode = jobData.nodeCount * ($scope.data.app.defaultProcessorsPerNode / $scope.data.app.defaultNodeCount);
+                jobData.processorsPerNode = jobData.nodeCount * ($scope.data.app.defaultProcessorsPerNode || 1) / ($scope.data.app.defaultNodeCount || 1);
```

Comment:
Does this need another set of parenthesis around `($scope.data.app.defaultProcessorsPerNode || 1) / ($scope.data.app.defaultNodeCount || 1)`?

Reply:
Nah, that extra set of parenthesis was redundant here since any order of operations will yield the same result